app/src/main/res/values-am-rET/strings.xml

```diff
@@ -561,4 +561,4 @@
 
```

Comment:
Making this change here means that we have broken the tests in earlier commits since they will have compilation issues. We will need to make these changes a part of the earlier commit where we changed the `PatientEntryValidationError` type to a sealed class.